version.go

```diff
@@ -11,7 +11,7 @@ import (
 var CurrentCommit string
 
 // CurrentVersionNumber is the current application's version literal
-const CurrentVersionNumber = "0.17.0"
+const CurrentVersionNumber = "v0.18.0-rc1"
```

Comment:
There shouldn't be a `v` here!

```suggestion
const CurrentVersionNumber = "0.18.0-rc1"
```